src/main/java/zoe/TaskList.java

```diff
@@ -0,0 +1,62 @@
+package zoe;
+
+import java.util.ArrayList;
+
+/**
+ * Stores all tasks that is manipulated by zoe
+ */
+public class TaskList {
+    protected ArrayList<Task> tasks;
+
+    public TaskList(ArrayList<Task> tl) {
+        this.tasks = tl;
+    }
+
+    /**
+     * Adds a new task to the list, when add(task) is called
+     * @param t
+     */
+    public void add(Task t) {
+        this.tasks.add(t);
+    }
+    /**
+     * Deletes a new task to the list, when delete(task) is called
+     * @param i
+     */
+    public void delete(int i) {
+            tasks.remove(Integer.valueOf(i) - 1);
+    }
+
+    /**
+     * Marks task at a given index when markAsDone(index) is called
+     * @param i
+     */
+    public void markAsDone(int i) {
+        tasks.get(i - 1).markAsDone();
+    }
+
+    /**
+     * Unmarks the task at a given index when unmark(index) is called
+     * @param i
+     */
+    public void unmark(int i) {
+        tasks.get(i - 1).unmark();
+    }
+    public ArrayList<Task> getTasks() {
+        return this.tasks;
+    }
+
+    public int getSize() {
+        return this.tasks.size();
+    }
+
+    public Task get(int i) {
+        return this.tasks.get(i - 1);
+    }
+
+    /**
+     * Checks if a task exists in the list given an index when isValid(index) is called.
```

Comment:
I noticed that there is no line spacing in between the description and parameter sections for your Javadocs comments, should you consider following the Java Coding Standard? 

One of the comment does not have a method associated with it, should you consider removing it?